### PR TITLE
Service

### PR DESCRIPTION
**Pull request by TehanovAnton**
Issue #9 
i create Service model and add associations

### CODE REVIEW

**Comment by dmokreckiy** [comment]
@TehanovAnton PR description - good
No reviewer and assignee.

File: app/controllers/omniauth_callbacks_controller.rb [hunk 1]
```diff
@@ -7,6 +7,7 @@ def google_oauth2
 
     if @user.persisted?
       flash[:notice] = I18n.t 'devise.omniauth_callbacks.success', kind: 'Google'
+      @user.confirm unless @user.confirmed?
```
Comment (dmokreckiy) [inline app/controllers/omniauth_callbacks_controller.rb]: Debugging?

File: app/controllers/cities_controller.rb [hunk 2]
```diff
@@ -0,0 +1,69 @@
+class CitiesController < ApplicationController
+  before_action :set_city, only: %i[ show edit update destroy ]
+
+  # GET /cities or /cities.json
```
Comment (dmokreckiy) [inline app/controllers/cities_controller.rb]: Comments

File: app/controllers/organizations_controller.rb [hunk 3]
```diff
@@ -0,0 +1,78 @@
+class OrganizationsController < ApplicationController
+  before_action :set_organization, only: %i[show edit update destroy]
+  before_action :set_cities, only: %i[new create edit update]
+
+  # GET /organizations or /organizations.json
```
Comment (dmokreckiy) [inline app/controllers/organizations_controller.rb]: Remove obsolete comments

File: app/controllers/organizations_controller.rb [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+  end
+
+  # GET /organizations/1/edit
+  def edit
+  end
+
+  # POST /organizations or /organizations.json
+  def create
+    @organization = Organization.new(organization_params)
+    saved = service.save(organization: @organization, city_ids: city_ids)
```
Comment (dmokreckiy) [inline app/controllers/organizations_controller.rb]: Check with previous PR about service

File: db/migrate/20211211194504_create_cities.rb [hunk 5]
```diff
@@ -1,7 +1,7 @@
 class CreateCities < ActiveRecord::Migration[6.1]
   def change
     create_table :cities do |t|
-      t.string :name
+      t.string :name, null: false
```
Comment (dmokreckiy) [inline db/migrate/20211211194504_create_cities.rb]: :+1:

File: db/migrate/20211231161011_create_services.rb [hunk 6]
```diff
@@ -0,0 +1,11 @@
+class CreateServices < ActiveRecord::Migration[6.1]
+  def change
+    create_table :services do |t|
+      t.string :name, null: false
+
```
Comment (dmokreckiy) [inline db/migrate/20211231161011_create_services.rb:8]: This should be bigger in field but will do for now

**Review by dmokreckiy — COMMENTED** [review]
Sync with previous PR.

**Review by yury-yaroshyk-itechart — APPROVED** [review]

**Pull request merged**